components/StatusIcon/constants.ts

```diff
@@ -0,0 +1,164 @@
+import { compileReactIconProperties } from '@cypress-design/react-icon'
```

Comment:
Importing react items in the common areas will bork up the typescript of TSX. You can experience it if you open a vue file and look at `class` or the same in react `className`.

Please move this in the react component.